_pages/acknowledgments.md

```diff
@@ -3,6 +3,22 @@
 
 # Acknowledgments
 
-This work has been supported by the [International Neuroinformatics Coordinating Facility](https://www.incf.org/){:target="_blank"}, the [Neuroimaging Data Sharing Task Force](https://web.archive.org/web/20170813183704/http://wiki.incf.org/mediawiki/index.php/Neuroimaging_Task_Force){:target="_blank"}, the Laura and John Arnold Foundation, NIMH Grant R24MH114705, and NSF grant OAC-1760950.
+This work has been done by the community of [individual contributors](https://bids-specification.readthedocs.io/en/stable/appendices/contributors.html). Their work was
+supported by the
+
+Funding Source | Target
+----- | -----
+[International Neuroinformatics Coordinating Facility](https://www.incf.org/){:target="_blank"} | [Neuroimaging Data Sharing Task Force](https://web.archive.org/web/20170813183704/http://wiki.incf.org/mediawiki/index.php/Neuroimaging_Task_Force){:target="_blank"} collated initial group of people to push pilot BIDS development and promotion
+Laura and John Arnold Foundation |
+[NIMH R24MH114705](https://reporter.nih.gov/project-details/9411944) | BIDS common derivatives BEP
+[NSF OAC-1760950](https://www.nsf.gov/awardsearch/showAward?AWD_ID=1760950) | 
```

Comment:
from the grant description it sounds like it also applies to common derivatives
```suggestion
[NSF OAC-1760950](https://www.nsf.gov/awardsearch/showAward?AWD_ID=1760950) | BIDS common derivatives BEP
```
am I correct @bendhouseart @poldrack ?

Reply:
I can't confirm or deny, @effigies recommended tagging Russ to confirm.

Reply:
yes, relevant to common derivatives